src/main/java/ch/uzh/ifi/hase/soprafs23/service/UserService.java

```diff
@@ -11,84 +11,75 @@
 import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.server.ResponseStatusException;
 
-import java.util.HashSet;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
 import java.util.Optional;
 import java.util.UUID;
 
-/**
- * User Service
- * This class is the "worker" and responsible for all functionality related to
- * the user
- * (e.g., it creates, modifies, deletes, finds). The result will be passed back
- * to the caller.
- */
 @Service
 @Transactional
 public class UserService {
 
-  //private final Logger log = LoggerFactory.getLogger(UserService.class); // unused
+    private final UserRepository userRepository;
 
-  private final UserRepository userRepository;
-
-  @Autowired
-  public UserService(@Qualifier("userRepository") UserRepository userRepository) {
-    this.userRepository = userRepository;
-  }
-
-  public List<User> getUsers() {
-    return this.userRepository.findAll();
-  }
-
-  public List getUsersFreeToInvite(){
-      // TODO: List with users which are not in a group yet
-      List L = new ArrayList<>();
-    return L;
-  }
-
-  // returns the userid of the user with the given token
-  // returns 0L if no user has the given token
-  public Long getUseridByToken(String token) {
-    if(token == null) {
-      return 0L;
+    @Autowired
+    public UserService(@Qualifier("userRepository") UserRepository userRepository) {
+        this.userRepository = userRepository;
     }
 
-    List<User> users = userRepository.findByToken(token);
-    assert users.size() <= 1; // if this is not the case, there is duplicate tokens!!!
-
-    if(users.size() == 1) {
-      return users.get(0).getId();
+    public List<User> getUsers() {
+        return this.userRepository.findAll();
     }
 
-    return 0L; 
-  }
-
-  // creates a new user, throws CONFLICT (409) if something goes wrong
-  public User createUser(User newUser) {
-    newUser.setToken(UUID.randomUUID().toString());
-    newUser.setStatus(UserStatus.ONLINE);
-
-    // check that the username is still free
-    checkIfUsernameExists(newUser.getUsername());
-
-    // check if username is only latin letters
-    if (!newUser.getUsername().matches("^[a-zA-Z]+$")) {
-      throw new ResponseStatusException(HttpStatus.CONFLICT, "The username should only contain Latin letters (a-z, A-Z)!");
+    // TODO: why does this exist? this is not functionality we are going to implement right?
```

Comment:
prbly also my fault as this is a leftover of my invitation logic I guess. Can be deleted